standup/settings.py

```diff
@@ -141,16 +141,35 @@ def path(*paths):
                       default='sqlite:///db.sqlite3')
 }
 
-CACHES = {
-    'default': config('CACHE_URL',
-                      parser=django_cache_url.parse,
-                      default='locmem:default')
-}
+memcachier_servers = config('MEMCACHIER_SERVERS', raise_error=False)
+if memcachier_servers:
+    # use django-pylibmc
+    CACHES = {
+        'default': {
+            'BACKEND': 'django_pylibmc.memcached.PyLibMCCache',
+            'LOCATION': memcachier_servers,
+            'TIMEOUT': 500,
+            'BINARY': True,
+            'USERNAME': config('MEMCACHIER_USERNAME'),
+            'PASSWORD': config('MEMCACHIER_PASSWORD'),
+            'OPTIONS': {  # Maps to pylibmc "behaviors"
+                'tcp_nodelay': True,
+                'ketama': True
+            }
+        }
+    }
+    SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
+else:
+    CACHES = {
+        'default': config('CACHE_URL', parser=django_cache_url.parse, default='locmem:default')
+    }
+
 if CACHES['default'].get('BACKEND', '') == 'django.core.cache.backends.locmem.LocMemCache':
     # If we're using locmem, set timeout for 60 seconds and restrict
     # cache to 100 things so as to not run rampant with memory usage.
     CACHES['default']['timeout'] = 60
     CACHES['default'].setdefault('OPTIONS', {})['MAX_ENTRIES'] = 100
```

Comment:
Pretty sure we can ditch this `if-clause`. I added it when we thought cache was related to our memory problems and that turned out not to be the case.